pyproject.toml

```diff
@@ -40,19 +40,20 @@ classifiers = [
 ]
 
 dependencies = [
-    "torchrl>=0.2.0",
-    "tensordict>=0.2.0",
-    "lightning>=2.1.0",
+    "einops",
     "hydra-core",
     "hydra-colorlog",
+    "lightning>=2.1.0",
+    "matplotlib",
     "omegaconf",
     "pyrootutils",
     "rich",
-    "einops",
-    "wandb",
-    "matplotlib",
+    "robust-downloader",
     "scipy",
-    "robust-downloader"
+    "tensordict>=0.2.0",
+    "torchrl>=0.2.0",
+    "vrplib>=1.3.0",
```

Comment:
(minor) should we include problem-specific dependencies as requirements, or is it better to have them as optional-dependencies?
CC @cbhua

Reply:
Just let me know if I should make any changes :)

Reply:
I would prefer to set these dependencies to optional. It's better to have fewer dependencies for the main RL4CO.  Also in the future, we will have more environments from various fields. They may bring quite a few dependencies.  We don't want these to be massive in the main RL4CO. 

In the meantime, we should write a clear notice to remind users to install problem-specific dependencies. In documents or environments README/comments.